src/components/Event.js

```diff
@@ -0,0 +1,140 @@
+import * as React from "react"
+
+import { createTheme, ThemeProvider } from "@mui/material"
+import cardex from "../assets/images/card-example.png"
+import Paper from "@mui/material/Paper"
+import Card from "@mui/material/Card"
+import CardHeader from "@mui/material/CardHeader"
+import CardMedia from "@mui/material/CardMedia"
+import CardContent from "@mui/material/CardContent"
+import CardActions from "@mui/material/CardActions"
+import Avatar from "@mui/material/Avatar"
+import Button from "@mui/material/Button"
+import Typography from "@mui/material/Typography"
+import Grid from "@mui/material/Grid"
+
+import EventAvailableIcon from "@mui/icons-material/EventAvailable"
+
+import YouTubeIcon from "@mui/icons-material/YouTube"
+import GitHubIcon from "@mui/icons-material/GitHub"
+import TwitterIcon from "@mui/icons-material/Twitter"
+
+const theme = createTheme({
+  spacing: [0, 4, 8, 16, 32, 64],
+  palette: {
+    primary: {
+      light: "#33cfd6",
+      main: "#00c4cc",
+      dark: "#00898e",
+      contrastText: "#fff",
+    },
+    secondary: {
+      light: "#d54b59",
+      main: "#cb1f30",
+      dark: "8e1521",
+      contrastText: "#fff",
+    },
+  },
+})
+
+const Event = () => {
```

Comment:
You could move the data into its own separate object outside of the component that way it would make it easier to migrate over to Meetups API. For example:

```js
const MOCK_EVENT_DATA = {
  presenter: "Fabian Buentello",
  title: "Some Presentation title Here",
  date: "July 14th, 2022 @ 11:00 AM CST",
  description: "<some long description here>",
  youtube: "http://link-to-youtube",
  meetup: "http://link-to-meetup",
  ...
}
const Event = (props = MOCK_EVENT_DATA) => {
  return (
    <ThemeProvider theme={theme}>
      <Grid...>
        ...
          <Grid item xs={12}>
            <Typography color="primary" variant="h3" mb={3} mt={3}>
              NEXT EVENT
            </Typography>
          </Grid>
          <Grid item xs={12}>
            <Card elevation={3} sx={{ minWidth: 300, maxWidth: 700 }}>
              <CardHeader
                ...
                title={props.presenter} // # here
                subheader={props.date} // # here
              />
              <CardMedia
                component="img"
                height="194"
                image={cardex}
                alt="Event Image or Title"
              />
              <CardContent>
                <Typography variant="h5" color="secondary">
                  {props.title} // # here
                </Typography>
                
                ...
      </Grid>
    </ThemeProvider>
  )
}

```

Reply:
Yep, that's the plan. I've looked at the Meetup.com documentation of what can be queried, but I want to see what that actually looks like when it pulls in to decide on whether the layout needs to change. But yes, the API call will be in its own component.